src/kioskboard.js

```diff
@@ -764,14 +781,14 @@
               var isBrowserEdgeLegacy = userAgent.indexOf('edge') > -1;
               var isBrowserInternetExplorer = userAgent.indexOf('.net4') > -1;
               var isBrowserEdgeWebView = isBrowserEdgeLegacy && userAgent.indexOf('webview') > -1;
-              
+
               if ((!isBrowserEdgeLegacy || isBrowserEdgeWebView) && !isBrowserInternetExplorer) {
                 var scrollTimeout = setTimeout(function () {
-                if (isBrowserEdgeWebView) {
-                  window.scrollBy(0, theInputOffsetTop);
-                } else {
-                  window.scrollTo({ top: scrollTop, left: 0, behavior: scrollBehavior });
-                }
```

Comment:
😊

Reply:
There were tab issues also, because it was modified in browser earlier. But now it is formatted by VS2022.

Reply:
@furcan  Would you like to rollback these changes?

Reply:
Hi @surexxx , no thank you, they can stay like that. In any case, I will be going to format the file before making a release. Thank you so much for your contributions.